README.md

```diff
@@ -7,31 +7,45 @@
 [![Crates.io](https://img.shields.io/crates/v/tplay)](https://crates.io/crates/tplay)
 [![Crates.io](https://img.shields.io/crates/d/tplay)](https://crates.io/crates/tplay)
 [![License](https://img.shields.io/badge/license-MIT-blue)](LICENSE)
+[![Aur](https://img.shields.io/aur/version/tplay-git)](https://aur.archlinux.org/packages/tplay-git)
```

Comment:
this is ok

Reply:
I think you can merge this now :smile: .